packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -264,16 +264,19 @@ export class MetriportMedicalApi {
    * @param dateFrom Optional start date that resources will be filtered by (inclusive). Format is YYYY-MM-DD.
    * @param dateTo Optional end date that resources will be filtered by (inclusive). Format is YYYY-MM-DD.
    * @param req.query.conversionType Optional to indicate how the medical record should be rendered.
+   * @param req.body Optional metadata to be sent through webhook
```

Comment:
This is SDK, there's no `req` param here. Can we update these docs (the new and the existing `req.query.conversionType` to represent the parameters, please?

Also, could add a bit more context to the param, maybe:

"Optional metadata to be sent along the webhook request as response of this query."